### PR TITLE
The CloudImage global config design

### PR DESCRIPTION
**Pull request by fanux**
For application configuration.

### CODE REVIEW

File: docs/design/global-config.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# Global configuration
+
+The feature of global configuration is to expose the parameters of distributed applications in the entire cluster mirror. 
+It is highly recommended to expose only a few parameters that users need to care about.
+
+If too many parameters need to be exposed, for example, the entire helm's values ​​want to be exposed, 
+then it is recommended to build a new image and put the configuration in to overwrite it.
+
+Using dashboard as an example, we made a cluster mirror of dashboard, 
+but different users want to use different port numbers when installing.
```
Comment (justadogistaken) [inline docs/design/global-config.md:10]: while installing

File: docs/design/global-config.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+To write kubefile, you need to copy yaml to the manifests directory at this time, 
+sealer only renders the files in this directory:
+
+```yaml
+FROM kubernetes:1.16.9
+COPY dashobard.yaml manifests/
+CMD kubectl apply -f manifests/dashobard.yaml
+```
+
+For users only need to specify the cluster environment variables:
```
Comment (justadogistaken) [inline docs/design/global-config.md:46]: For users, they only ....

File: docs/design/global-config.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+apiVersion: v1
+metadata:
+  labels:
+    k8s-app: kubernetes-dashboard
+  name: kubernetes-dashboard
+  namespace: kubernetes-dashboard
+spec:
+  ports:
+    -port: 443
+      targetPort: {{ DashBoardPort }}
```
Comment (justadogistaken) [inline docs/design/global-config.md:31]: cut a space

File: docs/design/global-config.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+```yaml
+FROM kubernetes:1.16.9
+COPY dashobard.yaml manifests/
+CMD kubectl apply -f manifests/dashobard.yaml
+```
+
+For users only need to specify the cluster environment variables:
+
+```shell script
+sealer run -e DashBoardPort=8443 mydashboard:latest
```
Comment (justadogistaken) [inline docs/design/global-config.md:49]: will {{ DashBoardPort }} go conflict with other value render, like helm? will it be better if define value like {{ Sealer-DashBoardPort }}  for env DashBoardPort (an example)

Comment (fanux) [reply docs/design/global-config.md:49]: This is up to the user.

**Review by justadogistaken — APPROVED** [review]
LGTM

**Pull request merged**